docs/dev/table/functions.md

```diff
@@ -1219,6 +1219,18 @@ TAN(numeric)
       </td>
     </tr>
 
+    <tr>
+      <td>
+        {% highlight text %}
+TANH(numeric)
+{% endhighlight %}
+      </td>
+      <td>
+        <p>Returns the hyperbolic tangent of <i>NUMERIC</i>.</p> 
+        <p>Return value type is <i>DOUBLE</i>.</p>
```

Comment:
Change the `NUMERIC` to lowercase here.

The return type is ...

Please ALWAYS be careful with these minor issues cause they have been pointed out many times.

Reply:
In fact, I just followed the [same pattern](https://github.com/apache/flink/pull/6700/files#diff-51c7d3bf736c60e8b942791a3715cbffR1284) as the PR of cosh, which looks good to Piotr, and according to Piotr's advice: [the type is best written uniformly](https://github.com/apache/flink/pull/6700#discussion_r220892667), and you haven't objected to this point before. But here you suggest I change it to lowercase. Your guidelines seem vague.

However, there is also my problem. I should have made these functions in the same PR to make the change more effective. There are a few points that I probably didn't pay much attention to, but I think many of the points you mentioned earlier are about documentation. Document description, however, has no definite criteria, and may be what you think is standard, but another person disagrees. So it caused a lot of reciprocating.

Plus, if you're willing to review, I think it's more contributor-friendly for you to give a definite review conclusion once (except for new problems arising from secondary changes). I feel that each time you give a partial review suggestion (some of which are not related to my last change), it also creates more reciprocity.

Reply:
Hi @yanghua, sorry for the inconvenience caused. First of all, I must admit that the case problem was carelessly omitted by me in your last PR, even though it had been pointed out. I'll fix it later.

About the documentation, yes, different contributors got different options on the standard style. That is why the documentation looked a mess in the past (the distributed contributions should not be an excuse) and it took me a lot of time unifying the format. IMO, the document style is even more important than the code style since it directly reflects the quality of a project, at least for beginners. Hopefully, you could understand.

All the PRs I've reviewed recently are about built-in functions and their's actually no big problem. However, even little changes can contain "bugs", which should be taken carefully. So, I wonder if you could be patient with that.

Best, Xingcan

Reply:
@xccui OK, I totally agree with you. 

In fact, I don't think your review suggestion is unreasonable. I just tried to explain, what are the reasons for the "ALWAYS" mentioned in your comment. In addition, I suggest that if the review process is more efficient, it will help me to adopt your suggestion (because you know that I am not focused on contributing built-in functions, and most of the time, I need to track a dozen opened PRs) .

Therefore, I suggest that the next time you feel that there is a problem, you directly point out what the problem is, that is enough.

Reply:
Will update the documentation.

Reply:
In my review I didn't focus as much on the style of the documentation, since I'm not familiar with it myself, especially once @xccui jumped in to this part I wanted to refer to his judgement there. Somehow this issue must have slipped our (mine and @xccui) attention that at the end it wasn't properly addressed in the `cosh` PR. Before merging I had an impression that @xccui comments were fully addressed, sorry if that wasn't the case.

btw, @yanghua I think you skipped two occurrences of `<i>NUMERIC</i>`. Here.

Reply:
@pnowojski Yes, in cosh PR, @xccui  has given this review suggestion. But because I submitted a lot of PRs, I forgot this suggestion, and you probably didn't pay attention to it (this is why I disscuss with xccui that if we review more efficiently and it would be easier to avoid this happening). In the end, I used that PR as a template.

I think this question has been 100% clear. Respect your and @xccui 's review suggestions. Thank you again for your time and effort.